src/deploy.sh

```diff
@@ -479,6 +487,29 @@ function modules_install_to()
   cmd_manager "$flag" "$cmd"
 }
 
+# This function will copy and install the compiled dtb files
+# @target Where should the dtb files be installed (1-vm, 2-local or 3-remote)
+function dtb_install()
```

Comment:
I'm wondering if this dtb install should be here... Maybe it makes more sense in the plugins folder `plugins/kernel_install/`?

Reply:
@rodrigosiqueira I don´t think so, the plugins directory is solely composed of scripts that will run in a remote machine while this function is called directly in the host. I think moving it there will just make things more confusing